Composer/packages/lib/code-editor/demo/src/multiEditors.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React, { useState, Fragment } from 'react';
+
+import { LgEditor } from '../../src';
+
+const content = `# Greeting1
+-Good morning
+
+# Greeting2
+-Good afternoon
+
+# Greeting3
+-Good evening
+`;
+
+function LGEditor() {
+  const [value, setValue] = useState(content);
+
+  const onChange = value => {
+    setValue(value);
+  };
+
+  const props = {
+    value,
+    onChange,
+    languageServer: {
+      host: 'localhost:5000',
+      path: '/lg-language-server',
+    },
+  };
+  return <LgEditor {...props} />;
+}
+
+export default function App() {
+  const editor0 = <LGEditor />;
+  const [editors, setEditors] = useState([editor0]);
```

Comment:
It sounds weird to save React component in an array and let it survive across multiple renders.